<?php

namespace ContainerRWxfJwK;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/*
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getAkeneoMeasurement_Validation_MeasurementFamily_OperationCountService extends KernelProdContainer
{
    /*
     * Gets the private 'akeneo_measurement.validation.measurement_family.operation_count' shared service.
     *
     * @return \Akeneo\Tool\Bundle\MeasureBundle\Validation\MeasurementFamily\OperationCountValidator
     */
    public static function do($container, $lazyLoad = true)
    {
        include_once \dirname(__DIR__, 4).'/vendor/symfony/validator/ConstraintValidatorInterface.php';
        include_once \dirname(__DIR__, 4).'/vendor/symfony/validator/ConstraintValidator.php';
        include_once \dirname(__DIR__, 4).'/vendor/akeneo/pim-community-dev/src/Akeneo/Tool/Bundle/MeasureBundle/Validation/MeasurementFamily/OperationCountValidator.php';

        return $container->privates['akeneo_measurement.validation.measurement_family.operation_count'] = new \Akeneo\Tool\Bundle\MeasureBundle\Validation\MeasurementFamily\OperationCountValidator(5);
    }
}
